src/_check.sh

```diff
@@ -32,15 +31,9 @@ function mod_check() {
 }
 function check_for_maintenance_mode() {
   title "Checking Maintenance Mode"
-  IN_MAINTENANCE="null"
-  IN_MAINTENANCE=$(curl -ks https://${CLIENT_DOMAIN_NAME}/api/v2/health/full | jq .data.inMaintenanceMode)
-  if [ $IN_MAINTENANCE == "true" ]; then
-    info "Maintenance Mode: $IN_MAINTENANCE"
-  elif [ $IN_MAINTENANCE == "false" ]; then
-    info "Maintenance Mode: $IN_MAINTENANCE"
-  else
-    info "Maintenance Mode: Error"
-  fi
+  IN_MAINTENANCE="Unknown"
+  IN_MAINTENANCE=$(curl -ks https://${CLIENT_DOMAIN_NAME}/api/v2/health/full | jq .data.inMaintenanceMode) || IN_MAINTENANCE="Unknown"
+  info "Maintenance Mode: $IN_MAINTENANCE"
```

Comment:
```suggestion
  IN_MAINTENANCE=$(curl -ks https://${CLIENT_DOMAIN_NAME}/api/v2/health/full | jq .data.inMaintenanceMode || "Unknown")
  info "Maintenance Mode: $IN_MAINTENANCE"
```

does this match as an equivalent?